src/main/java/onlinebookstoreapp/service/impl/BookServiceImpl.java

```diff
@@ -10,19 +14,31 @@
 @Service
 public class BookServiceImpl implements BookService {
     private final BookRepository bookRepository;
+    private final BookMapper bookMapper;
 
     @Autowired
-    public BookServiceImpl(BookRepository bookRepository) {
+    public BookServiceImpl(BookRepository bookRepository, BookMapper bookMapper) {
         this.bookRepository = bookRepository;
+        this.bookMapper = bookMapper;
     }
```

Comment:
Speaking of @RequiredArgsConstructor: that's why you wont be bothered changing constructor in the future, when you'll be adding more and more fields.